frontend/src/components/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,92 @@
+import { FC } from 'react';
+
+import { CategoriesList } from './components/categories-list/categories-list';
+import styles from './styles.module.scss';
+
+const Dashboard: FC = () => {
```

Comment:
you should to take this data from DB

Reply:
you should do it in this task

Reply:
Do you mean the frontend part with store?

Reply:
right